src/ir/parse.cpp

```
@@ -171,6 +173,13 @@ Instruction parse_instruction(const std::string& line, const std::map<std::strin
     if (regex_match(text, m, regex("callx " REG))) {
         return Callx{reg(m[1])};
     }
+    if (regex_match(text, m, regex("call_btf " FUNC R"_(\s+module\s+)_" IMM))) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)